code/modules/clothing/head/jobs.dm

```diff
@@ -247,17 +247,17 @@
 	. = ..()
 	var/mob/living/carbon/wearer = loc
 	if(!istype(wearer) || speaker != wearer) //if we are worn
-		return FALSE
+		return .
```

Comment:
this isnt necessary AND if youre still going to do that
```suggestion
		return
```
since it would be functionally the same anyway
you can read up on that in 7.2 of http://www.byond.com/docs/guide/chap06.html